### PR TITLE
kms integration & managed rules on default security groups

### PR DESCRIPTION
**Pull request by confusdcodr**

### CODE REVIEW

**Comment by confusdcodr** [comment]
@lorengordon added variables for ingress/egress rules

File: tardigrade/roots/aws/baseline/variables.tf [hunk 1]
```diff
@@ -113,3 +113,40 @@ variable "config_bucket_versioning" {
   description = "Whether versioning is enabled on parent config bucket"
   default     = false
 }
+
+##### SECURITY GROUP VARIABLES #####
+variable "default_vpc_sg_ingress_rules" {
+  description = "A schema list of ingress rules for the Defaul VPC's Default Security Group, see https://www.terraform.io/docs/providers/aws/r/security_group.html#ingress"
```
Comment (lorengordon) [inline tardigrade/roots/aws/baseline/variables.tf]:
```suggestion
  description = "A schema list of ingress rules for the Default VPC's Default Security Group, see https://www.terraform.io/docs/providers/aws/r/security_group.html#ingress"
```

File: tardigrade/roots/aws/baseline/variables.tf [hunk 2]
```diff
@@ -113,3 +113,40 @@ variable "config_bucket_versioning" {
...
+  type        = list
+  default     = []
+}
+
+variable "default_vpc_sg_egress_rules" {
+  description = "A schema list of egress rules for the Defaul VPC's Default Security Group, see https://www.terraform.io/docs/providers/aws/r/security_group.html#egress"
```
Comment (lorengordon) [inline tardigrade/roots/aws/baseline/variables.tf]:
```suggestion
  description = "A schema list of egress rules for the Default VPC's Default Security Group, see https://www.terraform.io/docs/providers/aws/r/security_group.html#egress"
```

File: tardigrade/roots/aws/baseline/variables.tf [hunk 3]
```diff
@@ -113,3 +113,40 @@ variable "config_bucket_versioning" {
...
+  type        = list
+  default     = []
+}
+
+variable "default_vpc_revoke_rules_on_delete" {
+  description = "Determines whether to forcibly remove rules when destroying the Defaul VPC's Default security group"
```
Comment (lorengordon) [inline tardigrade/roots/aws/baseline/variables.tf]:
```suggestion
  description = "Determines whether to forcibly remove rules when destroying the Default VPC's Default security group"
```

**Pull request merged**